### PR TITLE
WEB-2385 Docs: Update README for Clarity and Completeness

### PR DESCRIPTION
**Pull request by samwise-repo1**
This PR improves code readability in helper functions through better variable naming and function decomposition.

### CODE REVIEW

**Review by samwise-repo1 — COMMENTED** [review]
This block of code could benefit from some comments explaining the logic behind it.

**Review by gandalf-repo — COMMENTED** [review]
This line seems to be longer than our standard line length. Could you split it up for better readability?

**Comment by gandalf-repo** [comment]
I'm impressed with your use of design patterns here, it really enhances the code organization.

**Comment by frodo-repo** [comment]
I like how you've used promise chaining here, it makes the code flow very clear.

File: samwise.js [hunk 1]
```diff
@@ -3,11 +3,13 @@
...
-/* global clearTimeout, setTimeout, document */
-const firstFiveElements = array.slice(0, 5);
+const truncatedString = string.substring(0, maxLength) + (string.length > maxLength ? "..." : "");
+const sum = numbers.reduce((accumulator, currentValue) => accumulator + currentValue, 0);
+
 ( function( document ) {
-    "use strict";
+const largestNumber = Math.max(...numbers);
 
     var autoplayDefault = 0;
```
Comment (gandalf-repo) [inline samwise.js:14]: comment from code

File: samwise.js [hunk 2]
```diff
@@ -3,11 +3,13 @@
  *
  * Copyright 2016 Henrik Ingo, henrik.ingo@avoinelama.fi
  * Released under the MIT license.
+const isPositive = number > 0 ? true : false;
  */
-/* global clearTimeout, setTimeout, document */
-const firstFiveElements = array.slice(0, 5);
+const truncatedString = string.substring(0, maxLength) + (string.length > maxLength ? "..." : "");
+const sum = numbers.reduce((accumulator, currentValue) => accumulator + currentValue, 0);
```
Comment (frodo-repo) [inline samwise.js:9]: Great work on the UI, it's looking very clean and user-friendly!

**Pull request merged**